packages/react-composites/src/composites/CallComposite/pages/ConfigurationPage.tsx

```diff
@@ -303,3 +268,181 @@ const localPreviewTrampoline = (mobileView: boolean, doNotShow?: boolean): JSX.E
   }
   return <LocalPreview mobileView={mobileView} showDevicesButton={mobileView} />;
 };
+
+const cameraPermissionGrantedTrampoline = (
+  cameraPermissionGranted: boolean | undefined,
+  videoState?: PermissionState | undefined
+): boolean | undefined => {
+  /* @conditional-compile-remove(call-readiness) */
+  return videoState ? videoState === 'granted' : cameraPermissionGranted;
+
+  return cameraPermissionGranted;
+};
+
+const micPermissionGrantedTrampoline = (
+  microphonePermissionGranted: boolean | undefined,
+  audioState?: PermissionState | undefined
+): boolean | undefined => {
+  /* @conditional-compile-remove(call-readiness) */
+  return audioState ? audioState === 'granted' : microphonePermissionGranted;
+
+  return microphonePermissionGranted;
+};
+
+/* @conditional-compile-remove(call-readiness) */
+const callReadinessModal = (
+  mobileView: boolean,
```

Comment:
when there's so many args passed into a function, consider using named args instead
i.e. fn({arg1: val, arg2: val, arg3: val) instead of fn(arg1, arg2, arg3)

Given this is a JSX Element best to make this a standard react component and update the signature to:
```tsx
const CallReadinessModal = (props: {...}) => JSX.Element
```

Reply:
made changes and pushed to PR https://github.com/Azure/communication-ui-library/pull/2529